Description: Fix hyphens used as minus in man page
Author: Daniel Schaal <farbing@web.de>
Forwarded: http://lists.nongnu.org/archive/html/bino-list/2011-10/msg00000.html
Applied-Upstream: http://git.sv.gnu.org/cgit/bino.git/commit/?id=b66c0e952bc85550eeccdbd8ad170c0444817c82
Last-Update: 2011-10-01

--- bino-1.2.0.orig/doc/bino.1
+++ bino-1.2.0/doc/bino.1
@@ -198,9 +198,9 @@ Set subtitle scale factor.
 .IP "\-\-subtitle\-color=\fICOLOR\fP"
 Set subtitle color, in [AA]RRGGBB format.
 .IP "\-\-subtitle\-parallax=\fIVAL\fP"
-Subtitle parallax adjustment (-1 to +1).
+Subtitle parallax adjustment (\-1 to +1).
 .IP "\-P|\-\-parallax=\fIVAL\fP"
-Parallax adjustment (-1 to +1).
+Parallax adjustment (\-1 to +1).
 .IP "\-C|\-\-crosstalk=\fIVAL\fP"
 Crosstalk leak level (0 to 1). Comma-separated values for the R,G,B channels.
 .IP "\-G|\-\-ghostbust=\fIVAL\fP"
